### PR TITLE
Terminology update and typo fixes

### PR DESCRIPTION
**Pull request by marians**

### CODE REVIEW

**Review by stone-z — APPROVED** [review]

**Comment by Oshratn** [comment]
Quick question while this is still open - does it make sense to add user questions at this point?

File: src/content/advanced/high-availability/control-plane/index.md [hunk 1]
```diff
@@ -37,7 +37,7 @@ As of workload cluster release v{{% first_aws_ha_controlplane_version %}} for AW
 Having multiple control plane nodes in different availability zones has several benefits:
 
 - **API downtimes during upgrades are reduced to a minimum**. With a single control plane node,
-  upgrading the cluster requires the only master to be terminated and rebooted with new
+  upgrading the cluster requires the only control plane node to be terminated and rebooted with a new
```
Comment (Oshratn) [inline src/content/advanced/high-availability/control-plane/index.md:40]:
```suggestion
  upgrading the cluster requires only the control plane node to be terminated and rebooted with a new
```

File: src/content/advanced/high-availability/control-plane/index.md [hunk 2]
```diff
@@ -71,21 +71,21 @@ the number of AZs in the region, the logic is:
...
 by applying the logic described above. Here, the AZ assignment of existing
 worker [node pools]({{< relref "/advanced/node-pools" >}}) is taken into account.
 
 ## Upgrades from previous releases {#upgrades}
 
-When upgrading a cluster to workload cluster release v11.4.0, the cluster will remain a single
-master cluster during and after the upgrade. The API unavailability during the
-upgrade that is typical for single master clusters will apply for this upgrade.
+When upgrading a cluster to workload cluster release v11.4.0 or later, the cluster will still
+have only one control plane node after the upgrade. The API unavailability during the
```
Comment (Oshratn) [inline src/content/advanced/high-availability/control-plane/index.md:82]:
```suggestion
only have one control plane node after the upgrade. The API unavailability during the
```

File: src/content/advanced/high-availability/control-plane/index.md [hunk 3]
```diff
@@ -71,21 +71,21 @@ the number of AZs in the region, the logic is:
...
+upgrade that is typical for a single control plane node will apply for this upgrade.
```
Comment (Oshratn) [inline src/content/advanced/high-availability/control-plane/index.md:83]:
```suggestion
upgrade that is expected for a single control plane node will apply for this upgrade.
```

File: src/content/advanced/high-availability/multi-az/index.md [hunk 4]
```diff
@@ -32,7 +32,7 @@ owner:
 
 With Giant Swarm on AWS and Azure you can easily launch clusters with worker nodes spread across multiple availability zones (AZ). This will lower the risk that your cluster will become unavailable due to an incident in a particular AWS or Azure data center.
 
-On AWS, starting with workload cluster release {{% first_aws_ha_controlplane_version %}}, control plane nodes of a cluster are spread over different availability zones by default, for high availability of the Kubernetes API. You can chose however to run only a single control plane node. Read [High availability Kubernetes masters]({{< relref "/advanced/high-availability/control-plane" >}}) for more information.
+On AWS, starting with workload cluster release {{% first_aws_ha_controlplane_version %}}, control plane nodes of a cluster are spread over different availability zones by default, for high availability of the Kubernetes API. You can chose however to run only a single control plane node. Read [High-availability Kubernetes control plane]({{< relref "/advanced/high-availability/control-plane" >}}) for more information.
```
Comment (Oshratn) [inline src/content/advanced/high-availability/multi-az/index.md:35]:
```suggestion
On AWS, starting with workload cluster release {{% first_aws_ha_controlplane_version %}}, control plane nodes of a cluster are spread over different availability zones by default, for high availability of the Kubernetes API. You can choose to run only a single control plane node. Read [High-availability Kubernetes control plane]({{< relref "/advanced/high-availability/control-plane" >}}) for more information.
```

**Review by Oshratn — CHANGES_REQUESTED** [review]
A few tiny edits to the English.

**Review by Oshratn — APPROVED** [review]

**Pull request merged**